pugdebug/debugger.py

```diff
@@ -28,6 +28,7 @@ class PugdebugDebugger(QObject):
     got_all_variables_signal = pyqtSignal(object)
     breakpoint_removed_signal = pyqtSignal(int)
     breakpoints_listed_signal = pyqtSignal(type([]))
+    expressions_evaluated_signal = pyqtSignal(type([]))
```

Comment:
Looking at https://github.com/robertbasic/pugdebug/pull/37/files#diff-76ff3069d35cf3567a6ad796ea558214R141 it returns a `dict`, but the definition of this `pyqtSignal` is that it will emit a `list` object. So change this to have `type({})`